### DISCUSSION TITLE
Token 2022 with Transfer hook - can't transfer using Phantom Wallet, can do it via code.

### DISCUSSION THREAD

**Discussion by inventionsbyhamid** [phantom/docs#154] [Help & Support] [answered]
### Summary

Hi, I created a Token 2022 standard token with transfer hook but Phantom fails to transfer normal transfer of token. Does Phantom not support Token 2022 standard tokens? If it supports is there limitations on extensions that are supported?
Because I am able to transfer successfully using code from Solana docs. I found a relevant issue as well which is similar to what I am facing.

https://github.com/solana-labs/solana-program-library/issues/6094

How to make transfer work for Token 2022 token with transfer extension in Phantom wallet?

### Steps to Reproduce

1. Create SPL token with Token 2022 standard with transfer extension, mint tokens
2. Try normal transfer of tokens from Phantom wallet
3. It shows unable to send in Phantom wallet.

### Phantom Version

23.19.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @inventionsbyhamid,

We currently do not support transfer hooks. Please keep an eye on our [release notes](https://github.com/orgs/phantom/discussions/categories/announcements) for future token extensions support.
